Common subdirectories: sed-4.1.5/config and sed-4.1.5-patched/config
Common subdirectories: sed-4.1.5/doc and sed-4.1.5-patched/doc
Common subdirectories: sed-4.1.5/intl and sed-4.1.5-patched/intl
Common subdirectories: sed-4.1.5/lib and sed-4.1.5-patched/lib
diff -c sed-4.1.5/lib/regcomp.c sed-4.1.5-patched/lib/regcomp.c
*** sed-4.1.5/lib/regcomp.c Tue Dec  6 08:46:51 2005
--- sed-4.1.5-patched/lib/regcomp.c Wed Sep 13 12:22:37 2006
***************
*** 904,910 ****
  
  static void
  internal_function
! init_word_char (re_dfa_t *dfa)
  {
    int i, j, ch;
    dfa->word_ops_used = 1;
--- 904,910 ----
  
  static void
  internal_function
! init_word_char (volatile re_dfa_t *dfa)
  {
    int i, j, ch;
    dfa->word_ops_used = 1;
diff -c sed-4.1.5/lib/regex_internal.h sed-4.1.5-patched/lib/regex_internal.h
*** sed-4.1.5/lib/regex_internal.h Tue Dec  6 08:50:56 2005
--- sed-4.1.5-patched/lib/regex_internal.h Wed Sep 13 12:24:26 2006
***************
*** 697,703 ****
  }
  
  static inline void
! bitset_mask (bitset_t dest, const bitset_t src)
  {
    int bitset_i;
    for (bitset_i = 0; bitset_i < BITSET_WORDS; ++bitset_i)
--- 697,703 ----
  }
  
  static inline void
! bitset_mask (volatile bitset_t dest, const bitset_t src)
  {
    int bitset_i;
    for (bitset_i = 0; bitset_i < BITSET_WORDS; ++bitset_i)
diff -c sed-4.1.5/lib/regexec.c sed-4.1.5-patched/lib/regexec.c
*** sed-4.1.5/lib/regexec.c Tue Dec  6 08:46:56 2005
--- sed-4.1.5-patched/lib/regexec.c Wed Sep 13 12:24:00 2006
***************
*** 17,23 ****
     License along with the GNU C Library; if not, write to the Free
     Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
     02111-1307 USA.  */
! 
  static reg_errcode_t match_ctx_init (re_match_context_t *cache, int eflags,
  				     int n) internal_function;
  static void match_ctx_clean (re_match_context_t *mctx) internal_function;
--- 17,25 ----
     License along with the GNU C Library; if not, write to the Free
     Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
     02111-1307 USA.  */
! typedef int bool;
! #define false (0)
! #define true (1)
  static reg_errcode_t match_ctx_init (re_match_context_t *cache, int eflags,
  				     int n) internal_function;
  static void match_ctx_clean (re_match_context_t *mctx) internal_function;
***************
*** 2894,2900 ****
  	      sizeof (re_dfastate_t *) * (path->alloc - old_alloc));
      }
  
!   str_idx = path->next_idx ?: top_str;
  
    /* Temporary modify MCTX.  */
    backup_state_log = mctx->state_log;
--- 2896,2902 ----
  	      sizeof (re_dfastate_t *) * (path->alloc - old_alloc));
      }
  
!   str_idx = path->next_idx ? path->next_idx : top_str;
  
    /* Temporary modify MCTX.  */
    backup_state_log = mctx->state_log;
Common subdirectories: sed-4.1.5/po and sed-4.1.5-patched/po
Common subdirectories: sed-4.1.5/sed and sed-4.1.5-patched/sed
diff -c sed-4.1.5/sed/fmt.c sed-4.1.5-patched/sed/fmt.c
*** sed-4.1.5/sed/fmt.c Tue Jun 21 14:09:47 2005
--- sed-4.1.5-patched/sed/fmt.c Wed Sep 13 12:20:02 2006
***************
*** 18,24 ****
  /* GNU fmt was written by Ross Paterson <rap@doc.ic.ac.uk>.  */
  
  #include "sed.h"
! 
  #include <stdio.h>
  #include <ctype.h>
  #include <sys/types.h>
--- 18,24 ----
  /* GNU fmt was written by Ross Paterson <rap@doc.ic.ac.uk>.  */
  
  #include "sed.h"
! #include <string.h>
  #include <stdio.h>
  #include <ctype.h>
  #include <sys/types.h>
diff -c sed-4.1.5/sed/sed.c sed-4.1.5-patched/sed/sed.c
*** sed-4.1.5/sed/sed.c Tue Jun 21 14:09:47 2005
--- sed-4.1.5-patched/sed/sed.c Wed Sep 13 12:19:37 2006
***************
*** 22,28 ****
  
  #include "sed.h"
  
! 
  #include <stdio.h>
  #ifdef HAVE_STRINGS_H
  # include <strings.h>
--- 22,28 ----
  
  #include "sed.h"
  
! #include <string.h>
  #include <stdio.h>
  #ifdef HAVE_STRINGS_H
  # include <strings.h>
Common subdirectories: sed-4.1.5/testsuite and sed-4.1.5-patched/testsuite
